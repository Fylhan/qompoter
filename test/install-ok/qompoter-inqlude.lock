{
  "name": "install-ok/qompoter-inqlude",
  "date": "2016-11-24T22:00:38+01:00",
  "require": {
    "laumaya/glc-lib": {"version": "master", "type": "git", "url": "", "md5sum": "e346149139660e833187ee08b3a0faef"},
    "cp/fylhan": {"version": "v1.1", "type": "qompoter-fs", "url": "", "md5sum": "9d57702586a06dcd5de4d717d936c9d6"},
    "kde/attica": {"version": "v5.25.0", "type": "git", "url": "", "md5sum": "a0b9fffa1fcb58b1c794198b42799c26"},
  }
}
