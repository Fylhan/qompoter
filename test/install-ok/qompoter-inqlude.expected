Qompoter
======== install

* kde/attica 5.25.0
  Downloading sources from Git...
  Warning: no 'qompoter.pri' found for this package
  done

* cp/fylhan v1.1
  Downloading sources...
  done

* digia/0.5.0 master
  Downloading sources from Git...
  Warning: no 'qompoter.pri' found for this package
  done

done
